react/fields/MonthPicker/NativeMonthPicker/NativeMonthPicker.less

```diff
@@ -16,13 +16,13 @@
   border-radius: @field-border-radius;
   color: @sk-charcoal;
   background-color: @sk-white;
+  appearance: none;
 
   &::-webkit-inner-spin-button,
   &::-webkit-calendar-picker-indicator,
   &::-webkit-clear-button {
     display: none;
     -webkit-appearance: none;
```

Comment:
It actually needed to be moved outside to deal with a separate issue on iOS. This combination of styles should be good now. The `appearance:none` takes care of the normal scenarios and the pseudo selectors deal with the extra webkit specific parts. Both are required though.